analytics.js

```diff
@@ -496,6 +496,17 @@ class Analytics {
     this.processTrack(event, properties, options, callback);
   }
 
+  /**
+   * A function to convert non-string IDs to string format
+   * @param {any} id
+   * @returns {string} id
+   */
+  getStringId(id) {
```

Comment:
This should be a util, not a class-level method.

Reply:
Addressed